app/pb_kits/playbook/pb_user_badge/docs/_user_badge_default.jsx

```diff
@@ -1,12 +1,32 @@
 import React from 'react'
 import { UserBadge } from '../../'
 
+const badge = (
+  <svg viewBox="0 0 200 250" preserveAspectRatio="xMidYMid meet">
+    <path
+        d="M198.4,71.5c0-0.2,0-0.4,0-0.5c-0.2-1.2-1.4-2.1-1.4-2.1L103.2,2.7c-0.7-0.4-1-0.6-1.6-0.6c-1.3,0-2.3,1-2.3,2.3c0,0.1,0,56.1,0,67.1h-0.6v29.6h99.8L198.4,71.5L198.4,71.5z"
+    />
+    <path
+        d="M172.9,159.2c-31.4,23-33.2,24.3-33.2,24.3l-40.5,3.2v-29.4L172.9,159.2z"
+    />
+    <path
+        d="M99.3,125.4V33.5L1.6,102.7l-0.1,0.1c-1,0.7-1.5,1.8-1.5,3v0v67.4v0c0,1.2,0.6,2.3,1.5,3l0.1,0.1l93.5,66.3c1.7,1.2,4.1,0,4.1-2.1v-28.9l40.5-28l-40.5,0.7v-25h73.7l25.4-16.9l0.2-16.8H99.3z"
+    />
+    <rect
+        height="24.3"
+        width="99.3"
+        x="99.3"
+        y="101.1"
+    />
+  </svg>
```

Comment:
This would be better if we can determine the best way to import the same SVG that we render as a partial on the Rails side.